components/scream/src/physics/p3/p3_functions.hpp

```diff
@@ -246,21 +246,24 @@ struct Functions
   // Calls polysvp1 to obtain the saturation vapor pressure, and then computes
   // and returns the saturation mixing ratio, with respect to either liquid or ice,
   // depending on value of 'ice'
-  KOKKOS_INLINE_FUNCTION
+  KOKKOS_FUNCTION
```

Comment:
@ambrad , are there potential performance implications for de-inlining these functions?

Reply:
Should be fine. We can de-inline a few small ones later during the optimization stage.